draft-packages/content-block/KaizenDraft/ContentBlock/ContentBlock.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react"
+import classNames from "classnames"
+import styles from "./styles.scss"
+
+export interface ContentBlockProps {
+  /**
+   * Uniquely identify this component for testing purposes
+   * @default ""
+   */
+  automationId?: string
+  children?: React.ReactNode
+
+  /**
+   * Not recommended. A short-circuit for overriding styles in a pinch
+   * @default ""
+   */
+  classNameAndIHaveSpokenToDST?: string
+
+  /**
+   * Not recommended. A short-circuit for overriding styles in a pinch
+   * @default ""
+   */
+  style?: React.CSSProperties
```

Comment:
I had forgotten I wrote that and meant to call it out as semi-controversial. My thought here is that there are going to be times when we want to apply specific style adjustments to a core building block in a way that _might_ not be possible with pure CSS — for example some dynamic calculation based on height, etc — and that having that as an out is a useful escape mechanism in those situations.

Reply:
@ActuallyACat — Did you have further feels about this?

Reply:
Got it - so `classNameAndIHaveSpokenToDST` = static style overrides, `style` = dynamic style overrides. I think that should be specifically called out in the docs. 

I am wondering if this is too "catch-all" and if it could be a little more specific. What kind of dynamic overrides can we reasonably anticipate? If that's too much of an unknown this approach seems reasonable to me.

Reply:
> I am wondering if this is too "catch-all" and if it could be a little more specific. What kind of dynamic overrides can we reasonably anticipate? If that's too much of an unknown this approach seems reasonable to me.

It is a _little_ tricky to anticipate I think. What we’d probably want to limit it to is any positional properties: margin, padding, position, top, left, etc.

Reply:
Something like this? 
`Pick<React.CSSProperties, "padding", "margin", "position", "top", "right", "bottom", "left">`

If you think other use cases might arise that are outside of these properties then what you already have is 💯

Reply:
The other applications I can think of here are transformations, so if we picked:

* padding (and all variants)
* margin (and all variants)
* position
* top
* right
* bottom
* left
* transform (and all variants)

Then that’d _probably_ account for the kinds of layout changes I can imagine wanting to apply dynamically.

Reply:
Have added these variants in now!

Reply:
Looking good!